File: ios/Classes/FlutterPluginAdhubPlugin.m
#import "FlutterPluginAdhubPlugin.h"
#import <AdHubSDK/AdHubSDK.h>
#import "FlutterPluginAdhubPluginInitialize.h"
#import "FlutterPluginAdhubRewardPlugin.h"
#import "FlutterPluginAdhubInterstitialPlugin.h"

@implementation FlutterPluginAdhubPlugin

+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
    [FlutterPluginAdhubPluginInitialize registerWithRegistrar:registrar];
    [FlutterPluginAdhubRewardPlugin registerWithRegistrar:registrar];
    [FlutterPluginAdhubInterstitialPlugin registerWithRegistrar:registrar];
}

@end

File: android/src/main/java/com/maoge/flutter_plugin_adhub/FlutterPluginAdhubPlugin.java
package com.maoge.flutter_plugin_adhub;


import io.flutter.plugin.common.MethodCall;
import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
import io.flutter.plugin.common.MethodChannel.Result;
import io.flutter.plugin.common.PluginRegistry.Registrar;


public class FlutterPluginAdhubPlugin implements MethodCallHandler {

    /** Plugin registration. */
    public static void registerWith(Registrar registrar) {
        FlutterPluginAdhubPluginInitialize.registerWith(registrar);
        FlutterPluginAdhubRewardPlugin.registerWith(registrar);
        FlutterPluginAdhubInterstitialPlugin.registerWith(registrar);
    }

    @Override
    public void onMethodCall(MethodCall call, Result result) {
    }

}
